metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -748,7 +753,7 @@ public RecordTemplate ingestAspectIfNotPresent(@Nonnull Urn urn, @Nonnull String
         long nextVersion = _aspectDao.getNextVersion(urnStr, aspectName);
 
         return ingestAspectToLocalDBNoTransaction(urn, aspectName, ignored -> newValue, auditStamp,
-            internalSystemMetadata, latest, nextVersion);
+            internalSystemMetadata, latest, nextVersion, null);
```

Comment:
Are we assuming some methods of ingestion are not going to pass these down ever? Does it not make sense for this flow?

Reply:
@RyanHolstien We did not implemente conditional updated for all flows, but just for few endpoints. For example, conditional update is not supported for GraphQL nor delete operation for example. Those two cases you have mentioned are not part of the flows we have choose to start from.